paxsenix/src/main/kotlin/com/metrolist/paxsenix/Paxsenix.kt

```
@@ -0,0 +1,463 @@
+        return scoredResults.take(5).map { (result, _) ->
+            var hasWordSync = false
+            var lyrics: String? = null
+            
+            try {
+                val (fetchedLyrics, wordSync) = fetchLyricsForTrackWithType(result.id)
+                if (fetchedLyrics.isNotEmpty()) {
+                    lyrics = fetchedLyrics
+                    hasWordSync = wordSync
+                }
+            } catch (e: Exception) {
+            }
```

detekt (1.23.8)

[warning] Empty catch block detected. If the exception can be safely ignored, name the exception according to one of the exemptions as per the configuration of this rule.

(detekt.empty-blocks.EmptyCatchBlock)